build.cmd

```diff
@@ -19,16 +19,18 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 rem copy nuget locally
-if not exist .nuget\NuGet.exe (
-  if not exist .nuget md .nuget
-  copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
-)
+if not exist .nuget md .nuget
+copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
 
 rem restore packages for build script
 .nuget\NuGet.exe restore .\packages.config -PackagesDirectory .\packages -Verbosity quiet
 
+rem find Visual Studio and MSBuild installation paths
+FOR /F "delims=" %%i IN ('.\packages\vswhere.1.0.62\tools\vswhere.exe -property installationPath') DO set VS_PATH=%%i

```

Comment:
Now that I think about it, there might be a better option. Csi.exe is included in the [Microsoft.Net.Compilers package](https://www.nuget.org/packages/Microsoft.Net.Compilers/), so we could use this instead of looking for csi.exe in the VS2017 installed. I just tried it, it works fine.